src/patternfly/components/ProgressStepper/progress-stepper.scss

```diff
@@ -413,6 +419,7 @@ $pf-c-progress-stepper--breakpoint-map: build-breakpoint-map();
   font-size: var(--pf-c-progress-stepper__step-description--FontSize);
   color: var(--pf-c-progress-stepper__step-description--Color);
   text-align: var(--pf-c-progress-stepper__step-description--TextAlign);
+  overflow-wrap: anywhere;
```

Comment:
do you need this? or could it inherit [overflow-wrap from __main](https://github.com/patternfly/patternfly/blob/c44c23f9dc3dd755d048793621ca56b88d654fb8/src/patternfly/components/ProgressStepper/progress-stepper.scss#L374)? also assuming `break-word` will work, but I think `anywhere` was only needed on the title because the button is finicky. Or we could update `__main` to use `anywhere`?

Reply:
👍  `overflow-wrap:anywhere` seems to play nicely with soft breaks so I'm putting it on `__main`. I updated the screen shots and included a hyphen to show wrapping.